lib/jormungandr/test/data/jormungandr/stake_pools/create-stake-pool.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Usage:
```

Comment:
Add a comment saying what the script is for.

"This script (re)generates data used by the stake pool unit tests. The generated keys and certificates are checked in to git/inlined in the spec sources."